### PR TITLE
docs: document viewer

### LINKED ISSUE #128: Documentation for the viewer
Add some documentation describing:
- [ ] the viewer's configuration (config.json)
- [ ] the UI itself and the different views
- [ ] the filter queries (exclusion, inclusion, filter, suggestions…)

Those should be added to the viewer manual page stub located in viewer/ldgallery-viewer.7.md, which will be shipped along with the viewer and the program.

### PR DESCRIPTION
**Pull request by pacien**
GitHub: closes #128

### CODE REVIEW

File: viewer/ldgallery-viewer.7.md [hunk 1]
```diff
@@ -1,37 +1,66 @@
...
+
+The web application runs in a JavaScript-enabled web browser and does not require any special software on the server's side.  Its responsiveness allows it to be used from either a desktop browser or some mobile device.
 
 
 # USER INTERFACE
 
-TODO
+The viewer's user interface is split into a main item view and a side search panel, which can be opened by using the appropriate button in the toolbar at the top-left corner.
+
+The former displays the gallery's directories and items as a thumbnail grid.
```
Comment (OzoneGrif) [inline viewer/ldgallery-viewer.7.md]: The main view displays the gallery's directories and items as a thumbnail grid.

File: viewer/ldgallery-viewer.7.md [hunk 2]
```diff
@@ -1,37 +1,66 @@
...
+
+The latter allows the user to interactively enter filtering queries to search through the gallery's content.  This panel features a list of relevant tags that can be used to build the search query.
```
Comment (OzoneGrif) [inline viewer/ldgallery-viewer.7.md]: The side panel allows the user to interactively enter filtering queries to search through the gallery's content. This panel features a list of relevant tags that can be used to build the search query.

File: viewer/ldgallery-viewer.7.md (hunk 1 above)
Comment (pacien) [reply viewer/ldgallery-viewer.7.md]: Fixed.

File: viewer/ldgallery-viewer.7.md (hunk 2 above)
Comment (pacien) [reply viewer/ldgallery-viewer.7.md]: Fixed.

**Pull request merged**